package.json

```
@@ -19,7 +19,9 @@
     "seed:run": "ts-node -r tsconfig-paths/register ./node_modules/typeorm-seeding/dist/cli.js seed",
     "schema:drop": "ts-node ./node_modules/typeorm/cli.js schema:drop",
     "schema:sync": "ts-node ./node_modules/typeorm/cli.js schema:sync",
-    "migration:generate": "yarn run typeorm migration:create -n"
+    "migration:generate": "yarn run typeorm migration:create -n",
```

If `run` is not needed can we remove it from here?

---

I think you can delete `yarn` also
```suggestion
    "migration:generate": "typeorm migration:create -n",
```

---

Also, you can use `p` (alias for --pretty flag) to apply multi-line formatting to your generated migration queries